elastic/kibana
# [Onboarding] Make search_indices index details page as default route in index management

## Summary
Makes `search_indices` index details page as default route in the index_management plugin list page. 


https://github.com/user-attachments/assets/65afec5c-733f-4657-a793-56e29c65cf11


**How to test:** 
1. Enable searchIndices plugin in `kibana.dev.yml` as this plugin is behind Feature flag 
```
xpack.searchIndices.enabled: true

```
2. [Create new index](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html)
3. Navigate to `index management` app
4. Click on index name and confirm is navigated to `/app/elasticsearch/indices/index_details/my-index/data`
5.  set `xpack.searchIndices.enabled: false` in `kibana.dev.yml` 
6. Navigate again to `index management` app
7.  Click on index name and confirm is navigated to index management index details page 

### Checklist

Delete any items that are not applicable to this PR.
